doc/source/whatsnew/v2.3.0.rst

```diff
@@ -34,7 +34,8 @@ Other enhancements
 
 - The semantics for the ``copy`` keyword in ``__array__`` methods (i.e. called
   when using ``np.array()`` or ``np.asarray()`` on pandas objects) has been
-  updated to raise FutureWarning with NumPy >= 2 (:issue:`60340`)
+  updated to work correctly with NumPy >= 2 (:issue:`57739`)
```

Comment:
```suggestion
  updated to raise FutureWarning with NumPy >= 2 (:issue:`60340`)
```

(I assume that is from a merge conflict that was resolved in the wrong direction)